css/app.css

```diff
@@ -417,10 +417,10 @@
 }
 
 .CodeMirror-hint-information .infoType {
-  color: #30a;
+  color: #CA9800;
   cursor: pointer;
   display: inline;
-  margin-right: 0.5em;
+  margin-right: 1ch;
```

Comment:
One thing to consider: `ch` is not super well supported in IE (http://caniuse.com/#search=ch).